client/components/overview/LovedFor.jsx

```diff
@@ -0,0 +1,17 @@
+// stretch feature
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const LovedFor = (props) => (
```

Comment:
No need for parentheses around props. ES6 fxns with only 1 parameter do not need parentheses.